wp-content/themes/hurumap/functions.php

```diff
@@ -209,5 +209,42 @@ function bidirectional_acf_update_value( $value, $post_id, $field  ) {
 function custom_index_name() {
     return 'outbreak';
 }
+function get_base_url() {
+    $value = get_fields( 'options' );
+if (empty($value) ){
+    return $value;
+}else{
+    return "value";
+}
+    // $front_end__options = get_option( 'front_end__option_name' ); // Array of All Options
+    // return $front_end__options['base_url_0']; // Base Url
+}
+
 
 add_filter( 'ep_index_name', 'custom_index_name');
+
+
+//  Add custom preview page url link
+  function custom_preview_page_link($link) {
+    $base_url = get_field( 'description','options' );
+
+    $parentId = wp_get_post_parent_id( get_the_id());
```

Comment:
`get_the_ID()` could return `false`

Reply:
@KhadijaMahanga  in what cases will it return false? and what is the foolproof way to check this?